internal/lookout/application.go

```diff
@@ -55,34 +54,20 @@ func StartUp(config configuration.LookoutConfiguration, healthChecks *health.Mul
 	var eventStream eventstream.EventStream
 
 	if !config.DisableEventProcessing {
-
-		if len(config.Jetstream.Servers) > 0 {
-			stream, err := eventstream.NewJetstreamEventStream(
-				&config.Jetstream,
-				jsm.SamplePercent(100),
-				jsm.StartWithLastReceived())
-			if err != nil {
-				panic(err)
-			}
-			eventStream = stream
-
-			healthChecks.Add(stream)
-		} else {
-			stanClient, err := eventstream.NewStanClientConnection(
-				config.Nats.ClusterID,
-				"armada-server-"+util.NewULID(),
-				config.Nats.Servers)
-			if err != nil {
-				panic(err)
-			}
-			eventStream = eventstream.NewStanEventStream(
-				config.Nats.Subject,
-				stanClient,
-				stan.SetManualAckMode(),
-				stan.StartWithLastReceived())
-
-			healthChecks.Add(stanClient)
+		stanClient, err := eventstream.NewStanClientConnection(
```

Comment:
can we remove this nats ingestion from lookout, since lookoutingester should handle via pulsar?

Reply:
we can- but I plan to do this in a follow up where we rip out large amounts of nats code